### DISCUSSION TITLE
Determine number of downloads

### DISCUSSION THREAD

**Discussion by plstw** [google/googletest#4540] [Community Help]
Hello everyone, we use google test in safety-related projects and I now have to argue proven in use for the version 1.8.1. Therefore I would need the number of downloads for this release. Does anyone know how I can find this out? Thank you!

**Comment by sykhro** [comment]
Have you tried looking at [github stats](https://github.com/google/googletest/pulse)?

**Reply by plstw** [reply]
Yes, I did, but couldn't find anything. My current research indicates that this information does not exist at all or only for the owner of the project.
